File: ProyectosJAVA/lp2softRRHHController/src/pe/edu/pucp/lp2soft/rrhh/dao/EmpleadoDAO.java

package pe.edu.pucp.lp2soft.rrhh.dao;

import java.util.ArrayList;
import pe.edu.pucp.lp2soft.rrhh.model.Empleado;

public interface EmpleadoDAO {
    public ArrayList<Empleado> listar();
    public int insertar(Empleado empleado);
    public int modificar(Empleado empleado);
    public int eliminar(int idEmpleado);
}

File: ProyectosNET/LP2soft/LP2softRRHHController/DAO/EmpleadoDAO.cs
using LP2softRRHHModel;
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace LP2softRRHHController.DAO
{
    public interface EmpleadoDAO
    {
        BindingList<Empleado> listar();
        int insertar(Empleado empleado);
        int modificar(Empleado empleado);
        int eliminar(int idEmpleado);
    }
}
